src/poetry/console/commands/cache/clear.py

```diff
@@ -12,7 +12,7 @@
 
 class CacheClearCommand(Command):
     name = "cache clear"
-    description = "Clears Poetry's cache."
+    description = "Clears a Poetry cache by name (see cache list)."
```

Comment:
```suggestion
    description = "Clears a Poetry cache by name."
```

I'm not a fan of this parenthetical, since we don't do anything like it anywhere else. Since this PR is related to the presentation during `poetry list`, it feels unnecessary as `cache list` will be right next to it, anyway.

Reply:
ok